### ISSUE TITLE
ERROR: this.finishPromiseResolver is not a function

### ISSUE THREAD

**Issue by Identity-labs** [dominique-mueller/angular-notifier#253]
Hi,

Using angular 14

This error is flooding our sentry project, i think some check is missing

`this.finishPromiseResolver is not a function`

**Comment by HungryProger** [comment]
Yes I have same on angular v15. How fix that? Or i should remove the plagin?

**Comment by Identity-labs** [comment]
@dominique-mueller 
Here you have repro
It's when notifier is used outside angular scope
https://stackblitz.com/edit/angular-notifier-demo-j3kvnt?file=src%2Fapp%2Fapp.component.ts
